tests/foreman/cli/test_discoveredhost.py

```diff
@@ -412,3 +414,35 @@ def test_positive_list_facts():
 
     :CaseImportance: High
     """
+
+
+@pytest.mark.tier1
+def test_positive_verify_updated_fdi_image(target_sat):
+    """Verify foreman-discovery-image is built on latest up-to-date RHEL
+
+    :id: 2ab2ad88-4470-4d4c-8e0b-5892ad8d688e
+
+    :steps:
+        1. Register Satellite to CDN and install foreman-discovery-image
+
+    :expectedresults: Installed foreman-discovery-image is built on latest up-to-date RHEL
+
+    Verifies: SAT-24197, SAT-25275
+
+    :BZ: 2271598
+
+    :customerscenario: true
+
+    :CaseImportance: Critical
+    """
+    discovery_ks_path = '/usr/share/foreman-discovery-image/foreman-discovery-image.ks'
+    target_sat.register_to_cdn()
+    target_sat.execute('yum -y --disableplugin=foreman-protector install foreman-discovery-image')
+
+    if target_sat.os_version.major == 9:
+        version = '8.9' if is_open('SAT-25275') else str(target_sat.os_version)
+    elif target_sat.os_version.major == 8:
+        version = '8.9' if is_open('SAT-24197') else str(target_sat.os_version)
```

Comment:
Not sure if it makes sense to verify that the incorrect version is present, shouldn't we query for the desired state and use blockedBy to get the test run once the bugs are closed?

Reply:
I considered this way to verify as there were two different issues for EL8 and EL9, where EL9 isn't expected to be fixed any time soon and EL8 issue is already in-progress but not sure when it'll be ready as well.
And incorrect version is not blocker here it works for both EL8 and EL9 right now, we're just adding this test to verify we build FDI always on latest RHEL, so I wouldn't prefer using blockedBy for this test

Reply:
I don't want to block this, but wouldn't this implementation create confusing messages? iiuc the Verifies tag will cause these jiras to get comments that the issue is verified by automation when this test passes, even before they are fixed :) @jameerpathan111 is that right?

Reply:
@pondrejk No, unless the issues mentioned by`Verifies` are in "Review" or "Release Pending" state, the comment for test results won't be added. In other words, by default, it only adds comment if the issue is fixed and is on_qa and not released yet.

Reply:
well, fair enough